.travis.yml

```diff
@@ -14,11 +14,11 @@ matrix:
         - PEP8=1
         - NUMPYSPEC=numpy
       before_install:
-        - pip install pep8==1.7.0
+        - pip install pycodestyle==1.7.0
```

Comment:
1.7.0 was only released for `pep8`, not `pycodestyle`. You'll need to use a newer version, such as 2.3.1